### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by Irostovsky** [MirkoMilanovic/Commission_Counter#1]
Hi @MirkoMilanovic 
I m Ivan from GuestReady.
Thanks for doing this task!

I have some notes about the current solution. Imagine that the date string which is passed to the function starts to be received from different sources with different formats. In reality, it is possible when you receive data from Airbnb and from booking.com.
For example:
from airbnb we receive date string as `2022-01-15`
but from booking.com as `12 June, 2023`
Please adopt the function to process return the commission by passed date string and platform(airbnb, bcom).
Thanks!

**Comment by MirkoMilanovic** [comment]
Hi @Irostovsky 

Yesterday I decided to make it very simple, but now I have finished the function more universal, with a platform, as suggested. 
Thank you for reaching me,
Mirko Milanovic

**Comment by Irostovsky** [comment]
Thanks @MirkoMilanovic 
It is much better. 
I just see 2 points:
1. Imagine that marketing team would set the commission for Dec to 11% and for Jan to 12%.
2. And imagine that there more platforms could be added in the future with different date formats.

How could you modify the code to be easily change the commission rates per month and add any platform without changing code logic much.
Thanks!

**Comment by MirkoMilanovic** [comment]
Thanks for the good points @Irostovsky !
Now, after the change, the code looks way better.

**Comment by Irostovsky** [comment]
Perfect work, @MirkoMilanovic .
I would just merge date_formats and platform_format. But it is small refacorign issue. No problem for now.
I will notify agency to organize the first interview with you. 
See you!

**Comment by MirkoMilanovic** [comment]
Yes, I did hesitate between those two... But anyway, thank you very much @Irostovsky .
See you, and have a nice day,
Mirko
